amulet/sentry.py

```diff
@@ -192,16 +192,16 @@ def relation(self, from_rel, to_rel):
 
 # Possibly use to build out instead of having code in setup()?
 class Talisman(object):
-    def __init__(self, services, rel_sentry='relation-sentry', juju_env=None, timeout=300):
```

Comment:
There really shouldn't be anything else passing relation-sentry into the method except what's in Amulet itself, since Talisman is an "internal" class.

I think we should just plan for Amulet 2.0, possibly starting at 1.8 branch on github to do these bug fixes to but start landing backwards incompatible changes to master